### ISSUE TITLE
passing a ref to Fade component throws a warning

### ISSUE THREAD

**Issue by Yuripetusko** [femioladeji/react-slideshow#192]
I am passing ref to a `Fade` like so:

```
const MyComponent = () => {
  const slideRef = useRef<Fade | null>(null);
  
  const onChange = () => {...}
  
  return (
     <Fade
        autoplay={false}
        duration={10000}
        arrows={false}
        ref={slideRef}
        onChange={onChange}
        transitionDuration={300}>
        {items.map((item) => (
          <Item
            slideIndicators={slideIndicators}
            item={item}
            key={item.id}
          />
        ))}
      </Fade>
  )
```

And getting a warning in browser console like so:

`Warning: [object Object]: `ref` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different **prop.`**

Not really sue where to look yet, but if there are any pointers happy to try to debug more

**Comment by femioladeji** [comment]
@Yuripetusko we'll look into this and get back to you

**Comment by femioladeji** [comment]
@Yuripetusko thanks for raising this issue. It has been fixed in the latest version `4.0.5`

**Comment by Yuripetusko** [comment]
Thank you for doing it so promptly
